### DISCUSSION TITLE
More options for the gauge panel

### DISCUSSION THREAD

**Discussion by wouter-heerwegh** [foxglove/community#27] [Q&A]
Hi,

I was thinking it could be nice to have more options for the gauge panel. Like for instance being able to set a title or something similar so that it's clear what the gauge is for. Also a possibility to show the current value which is displayed on the gauge. For instance something like this:
![image](https://user-images.githubusercontent.com/20578313/196953237-40320839-db11-4853-820b-ef8f49e2fbd2.png)

If you have some other ideas, feel free to add them aswell.

**Comment by jtbandes** [comment]
Thanks for the feedback! FYI, some similar suggestions have been discussed at:
- #4475
- #4394
- #4174

**Reply by horverno** [reply]
Also an inverted direction would be really helpful if we change the gauge minimum value to a bigger number than maximum. Thanks for considering!
